src/components/Header.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect, useState } from "react";
+import { getUserFolder, getUserId } from "../api.js";
+import "./Header.css";
+export default Header;
+function Header() {
+  return (
+    <header>
+      <nav>
+        <div className="nav-contents">
+          <a href="./folder">
+            <img
+              className="header-logo-img"
+              src="/imgs/logo.png"
+              alt="코드잇로고"
+            ></img>
+          </a>
+          <UserProfileInHeader />
+        </div>
+      </nav>
+      <UserFolderProfile />
+    </header>
+  );
+}
+
+function UserProfileInHeader() {
+  const [user, setUser] = useState(null);
+  const loginImg = <img src="/imgs/로그인.png" alt="로그인"></img>;
+  useEffect(() => {
+    async function handleLoginProfile() {
+      const user = await getUserId();
+      setUser(user);
+    }
+    handleLoginProfile();
+  }, []);
+  return (
+    <a href="./folder" className="user-profile-login">
+      {user ? (
+        <>
+          <img
+            className="user-profile-login-img"
+            src={user.profileImageSource}
+            alt="사용자 이미지"
+          ></img>
```

Comment:
[중요도: 중간] React의 경우 여러분들의 기존과 하던 순수 html과 다르게 굳이굳이 element를 닫지 않아도 됩니다.
```suggestion
          <img
            className="user-profile-login-img"
            src={user.profileImageSource}
            alt="사용자 이미지"
          />
```

Reply:
https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/pull/165/commits/263d22342bbb797183ed1575c371f278fb83f1dd

이미지 태그 끝을 수정했고 변수에 이미지를 담지 않고 바로 return에 이미지 태그를 넣었습니다.